### ISSUE TITLE
Keyboard Shortcuts Support

### ISSUE THREAD

**Issue by ohkimur** [DWilliames/idasen-controller-mac#7]
At the moment there is no support for keyboard shortcuts. I think that would be a nice addition to this app. I love how minimalist it is at the moment, but I would love to be able to take control of the desk using some nice keyboard shortcuts.

**Comment by agestaun** [comment]
Did you try with the system preferences option to create shorcuts?

**Comment by user928** [comment]
+1 for this

Shortcuts would avoid the need for apple script/better touch tool/alfred/raycast support

(you can't add this via system preferences since this app doesn't have any menu bars)

**Comment by ohkimur** [comment]
> Did you try with the system preferences option to create shorcuts?

Yes, I did. It actually works very nicely, especially with Raycast, but I still think it'll be cool to have this native in the app, at least as an option.
